### DISCUSSION TITLE
SWI-Prolog reasoner

### DISCUSSION THREAD

**Discussion by crapo** [SemanticApplicationDesignLanguage/sadl#588] [General]
@agabaldon , I was playing around with the SWI-Prolog reasoner/translator pair. I'm seeing things that seem too broken to have worked in ASKE. Do you have fixes to these plug-ins that are not in the development branch? I'd like to start from the latest as I make them more robust and improve the documentation on how to use them....

**Comment by agabaldon** [comment]
I don't think so. I think eventually any changes were merged into the main branches.
